libraries/tools/kotlin-stdlib-gen/src/templates/Mapping.kt

```diff
@@ -314,6 +314,7 @@ object Mapping : TemplateGroupBase() {
         specialFor(ArraysOfUnsigned) { inlineOnly() }
 
         doc { "Appends all elements yielded from results of [transform] function being invoked on each ${f.element} of original ${f.collection}, to the given [destination]." }
+        sample("samples.collections.Collections.Transformations.flatMap")
```

Comment:
The sample is for `flatMap` function, but it's placed into `flatMap`'s function template. 
But no worries: I'll fix it before merging.